src/styles/builder.scss

```diff
@@ -161,6 +165,20 @@ $headerHeight: 170px;
       > li {
         margin: 5px 0;
 
+        &.diet-item {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+
+          & .diet-item-cross {
```

Comment:
I was thinking about using bichromatic icons like the toggler because the contrast between green and white is not as good as I expected.
What do you think about something like this?
![image](https://user-images.githubusercontent.com/6814061/80923102-af537000-8d81-11ea-9d4b-8d87e4e0e12e.png)

Reply:
I like it. DONE in last commit